### ISSUE TITLE
Dependencies not released

### ISSUE THREAD

**Issue by gzoritchak** [JetBrains/xodus-dnq#59]
It's not possible to use the current version of the project. 

Could you release all the needed components?

```
> Could not find org.jetbrains.xodus:xodus-query:1.2.3217.
     Searched in the following locations:
       - https://jcenter.bintray.com/org/jetbrains/xodus/xodus-query/1.2.3217/xodus-query-1.2.3217.pom
       - https://jcenter.bintray.com/org/jetbrains/xodus/xodus-query/1.2.3217/xodus-query-1.2.3217.jar
       - http://kotlin.bintray.com/ktor/org/jetbrains/xodus/xodus-query/1.2.3217/xodus-query-1.2.3217.pom
       - http://kotlin.bintray.com/ktor/org/jetbrains/xodus/xodus-query/1.2.3217/xodus-query-1.2.3217.jar
       - https://dl.bintray.com/kotlin/kotlin-eap/org/jetbrains/xodus/xodus-query/1.2.3217/xodus-query-1.2.3217.pom
       - https://dl.bintray.com/kotlin/kotlin-eap/org/jetbrains/xodus/xodus-query/1.2.3217/xodus-query-1.2.3217.jar
       - https://repo.maven.apache.org/maven2/org/jetbrains/xodus/xodus-query/1.2.3217/xodus-query-1.2.3217.pom
       - https://repo.maven.apache.org/maven2/org/jetbrains/xodus/xodus-query/1.2.3217/xodus-query-1.2.3217.jar
     Required by:
         project : > org.jetbrains.xodus:dnq:1.1.363
         project : > org.jetbrains.xodus:dnq:1.1.363 > org.jetbrains.xodus:dnq-open-api:1.1.363
   > Could not find org.jetbrains.xodus:xodus-utils:1.2.3217.

```

**Comment by mazine** [comment]
@morj Could you please handle deps release as well?

**Comment by morj** [comment]
@gzoritchak as a workaround you can add our https://jetbrains.bintray.com/xodus repo which contains all needed artifacts, but we'll try to bring newest xodus from maven-central as well

**Comment by morj** [comment]
@gzoritchak fixed build 1.2.420 should be available now
